### PR TITLE
Add cron for decidim_initiatives tasks

### PR DESCRIPTION
**Pull request by LauraJaimeC**

### CODE REVIEW

File: app/scheduled_workers/decidim_initiatives.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DecidimInitiatives
+  include Sidekiq::Worker
+
+  def perform
+    system("bundle exec rake decidim_initiatives:check_validating")
+    system("bundle exec rake decidim_initiatives:check_published")
+    system("bundle exec rake decidim_initiatives:notify_progress")
```
Comment (tramuntanal) [inline app/scheduled_workers/decidim_initiatives.rb:9]: We would like to have one worker per rake task that way they are 100% independent in front of failure.

**Review by tramuntanal — CHANGES_REQUESTED** [review]
We need to have each rake task scheduled at a different time please

File: app/scheduled_workers/decidim_initiatives.rb (hunk 1 above)
Comment (LauraJaimeC) [reply app/scheduled_workers/decidim_initiatives.rb]: Can you review it now, please?

**Review by tramuntanal — APPROVED** [review]
:+1:

File: app/scheduled_workers/decidim_initiatives_notify_progress.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DecidimInitiativesNotifyProgress
+  include Sidekiq::Worker
+
+  def perform
+    system("bundle exec rake decidim_initiatives:notify_progress")
```
Comment (tramuntanal) [inline app/scheduled_workers/decidim_initiatives_notify_progress.rb:7]:
```suggestion
    system("bundle exec rake decidim_initiatives:notify_progress") if Rails.env.production?
```

**Review by tramuntanal — APPROVED** [review]
Good!

**Pull request merged**